IdentityCore/src/util/ASAuthorizationSingleSignOnProvider+MSIDExtensions.m

```diff
@@ -48,6 +48,7 @@ - (ASAuthorizationSingleSignOnRequest *)createSSORequestWithOperationRequest:(MS
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->